From d6f16e59e2f4b10d57bb9747032b7c094dc70813 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Date: Mon, 27 Aug 2018 13:22:34 +0530
Subject: [PATCH 268/741] bus: fsl-mc: DMA configure to have force-dma as
 default

commit 6920803941909ed0d7e2ce8be4ac2f8b2e06d2b8 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Implicit dma setting from bus works when dma-ranges
specified but not otherwise. We need to continue
to go for force_dma as default for cases dma-ranges not
specified. Example dynamic device tree generation for
generic kvm virtual machines.

Signed-off-by: Bharat Bhushan <Bharat.Bhushan@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/bus/fsl-mc/fsl-mc-bus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bus/fsl-mc/fsl-mc-bus.c b/drivers/bus/fsl-mc/fsl-mc-bus.c
index ae1cceea4a2a..8f504a220814 100644
--- a/drivers/bus/fsl-mc/fsl-mc-bus.c
+++ b/drivers/bus/fsl-mc/fsl-mc-bus.c
@@ -140,7 +140,7 @@ static int fsl_mc_dma_configure(struct device *dev)
 	while (dev_is_fsl_mc(dma_dev))
 		dma_dev = dma_dev->parent;
 
-	return of_dma_configure(dev, dma_dev->of_node, 0);
+	return of_dma_configure(dev, dma_dev->of_node, 1);
 }
 
 static ssize_t modalias_show(struct device *dev, struct device_attribute *attr,
-- 
2.17.1

